[2022-02-03 18:14:27,903] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-03 18:14:27,928] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-03 18:14:27,928] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 18:14:27,928] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-03 18:14:27,928] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 18:14:27,951] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 00:00:00+00:00
[2022-02-03 18:14:27,958] {standard_task_runner.py:52} INFO - Started process 877 to run task
[2022-02-03 18:14:27,962] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fhv_gcs_dag', 'format_to_parquet_task', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/fhv_gcs_dag.py', '--cfg-path', '/tmp/tmp3igh397q', '--error-file', '/tmp/tmpi0hiwlbk']
[2022-02-03 18:14:27,963] {standard_task_runner.py:77} INFO - Job 155: Subtask format_to_parquet_task
[2022-02-03 18:14:28,062] {logging_mixin.py:109} INFO - Running <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [running]> on host bd2f56b76e32
[2022-02-03 18:14:28,212] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 18:14:28,305] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhv_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T00:00:00+00:00
[2022-02-03 18:14:29,367] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 18:14:29,390] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=fhv_gcs_dag, task_id=format_to_parquet_task, execution_date=20190302T000000, start_date=20220203T181427, end_date=20220203T181429
[2022-02-03 18:14:29,500] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 18:14:29,620] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-19 23:38:20,204] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-19 23:38:20,232] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-19 23:38:20,233] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 23:38:20,233] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-19 23:38:20,233] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 23:38:20,263] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 00:00:00+00:00
[2022-02-19 23:38:20,274] {standard_task_runner.py:52} INFO - Started process 347 to run task
[2022-02-19 23:38:20,279] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fhv_gcs_dag', 'format_to_parquet_task', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '227', '--raw', '--subdir', 'DAGS_FOLDER/fhv_gcs_dag.py', '--cfg-path', '/tmp/tmp3t9cbi5t', '--error-file', '/tmp/tmpuc_jcn8g']
[2022-02-19 23:38:20,281] {standard_task_runner.py:77} INFO - Job 227: Subtask format_to_parquet_task
[2022-02-19 23:38:20,439] {logging_mixin.py:109} INFO - Running <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [running]> on host 812dd7ecce85
[2022-02-19 23:38:20,542] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-19 23:38:20,587] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhv_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T00:00:00+00:00
[2022-02-19 23:38:21,005] {taskinstance.py:1700} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 185, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/fhv_gcs_dag.py", line 27, in format_to_parquet
    table = pv.read_csv(src_file)
  File "pyarrow/_csv.pyx", line 867, in pyarrow._csv.read_csv
  File "pyarrow/_csv.pyx", line 876, in pyarrow._csv.read_csv
  File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 99, in pyarrow.lib.check_status
pyarrow.lib.ArrowInvalid: CSV parse error: Expected 6 columns, got 18: 1,2019-03-01 01:15:31,2019-03-01 01:29:24,1,1.40,1,N,158,79,1,10,3,0.5,4.1,0,0.3,17.9,2.5
[2022-02-19 23:38:21,035] {taskinstance.py:1277} INFO - Marking task as UP_FOR_RETRY. dag_id=fhv_gcs_dag, task_id=format_to_parquet_task, execution_date=20190302T000000, start_date=20220219T233820, end_date=20220219T233821
[2022-02-19 23:38:21,071] {standard_task_runner.py:92} ERROR - Failed to execute job 227 for task format_to_parquet_task
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 185, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/fhv_gcs_dag.py", line 27, in format_to_parquet
    table = pv.read_csv(src_file)
  File "pyarrow/_csv.pyx", line 867, in pyarrow._csv.read_csv
  File "pyarrow/_csv.pyx", line 876, in pyarrow._csv.read_csv
  File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 99, in pyarrow.lib.check_status
pyarrow.lib.ArrowInvalid: CSV parse error: Expected 6 columns, got 18: 1,2019-03-01 01:15:31,2019-03-01 01:29:24,1,1.40,1,N,158,79,1,10,3,0.5,4.1,0,0.3,17.9,2.5
[2022-02-19 23:38:21,106] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-02-19 23:38:21,300] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-23 19:18:06,194] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-23 19:18:06,214] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-23 19:18:06,214] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-23 19:18:06,214] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-23 19:18:06,214] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-23 19:18:06,234] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 00:00:00+00:00
[2022-02-23 19:18:06,241] {standard_task_runner.py:52} INFO - Started process 106 to run task
[2022-02-23 19:18:06,246] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fhv_gcs_dag', 'format_to_parquet_task', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '623', '--raw', '--subdir', 'DAGS_FOLDER/fhv_gcs_dag.py', '--cfg-path', '/tmp/tmpt3s6ixey', '--error-file', '/tmp/tmp7ouwtynv']
[2022-02-23 19:18:06,247] {standard_task_runner.py:77} INFO - Job 623: Subtask format_to_parquet_task
[2022-02-23 19:18:06,321] {logging_mixin.py:109} INFO - Running <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [running]> on host 537c9393f0b3
[2022-02-23 19:18:06,373] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-23 19:18:06,395] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhv_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T00:00:00+00:00
[2022-02-23 19:18:07,022] {python.py:175} INFO - Done. Returned value was: None
[2022-02-23 19:18:07,037] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=fhv_gcs_dag, task_id=format_to_parquet_task, execution_date=20190302T000000, start_date=20220223T191806, end_date=20220223T191807
[2022-02-23 19:18:07,101] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-23 19:18:07,155] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-23 20:04:01,269] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-23 20:04:01,330] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-23 20:04:01,331] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-23 20:04:01,331] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-23 20:04:01,331] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-23 20:04:01,414] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 00:00:00+00:00
[2022-02-23 20:04:01,442] {standard_task_runner.py:52} INFO - Started process 138 to run task
[2022-02-23 20:04:01,458] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fhv_gcs_dag', 'format_to_parquet_task', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/fhv_gcs_dag.py', '--cfg-path', '/tmp/tmpa5hjmo1x', '--error-file', '/tmp/tmpf71grr6p']
[2022-02-23 20:04:01,459] {standard_task_runner.py:77} INFO - Job 6: Subtask format_to_parquet_task
[2022-02-23 20:04:01,642] {logging_mixin.py:109} INFO - Running <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [running]> on host 1bb54b80b2ee
[2022-02-23 20:04:01,725] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-23 20:04:01,780] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhv_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T00:00:00+00:00
[2022-02-23 20:04:04,537] {python.py:175} INFO - Done. Returned value was: None
[2022-02-23 20:04:04,623] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=fhv_gcs_dag, task_id=format_to_parquet_task, execution_date=20190302T000000, start_date=20220223T200401, end_date=20220223T200404
[2022-02-23 20:04:05,120] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-23 20:04:05,463] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-23 20:39:58,910] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-23 20:39:58,924] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [queued]>
[2022-02-23 20:39:58,924] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-23 20:39:58,924] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-23 20:39:58,925] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-23 20:39:58,944] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2019-03-02 00:00:00+00:00
[2022-02-23 20:39:58,952] {standard_task_runner.py:52} INFO - Started process 206 to run task
[2022-02-23 20:39:58,961] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'fhv_gcs_dag', 'format_to_parquet_task', 'scheduled__2019-03-02T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/fhv_gcs_dag.py', '--cfg-path', '/tmp/tmphubupyn9', '--error-file', '/tmp/tmpugmhgdsf']
[2022-02-23 20:39:58,962] {standard_task_runner.py:77} INFO - Job 6: Subtask format_to_parquet_task
[2022-02-23 20:39:59,038] {logging_mixin.py:109} INFO - Running <TaskInstance: fhv_gcs_dag.format_to_parquet_task scheduled__2019-03-02T00:00:00+00:00 [running]> on host 1c172e0325c8
[2022-02-23 20:39:59,107] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-23 20:39:59,153] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=fhv_gcs_dag
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T00:00:00+00:00
[2022-02-23 20:40:00,538] {python.py:175} INFO - Done. Returned value was: None
[2022-02-23 20:40:00,603] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=fhv_gcs_dag, task_id=format_to_parquet_task, execution_date=20190302T000000, start_date=20220223T203958, end_date=20220223T204000
[2022-02-23 20:40:00,711] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-23 20:40:00,772] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
